<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>T-Card</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lNC_2">
       <property name="text">
        <string>Лог</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="consol">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Операции с картой</string>
     </attribute>
     <widget class="QGroupBox" name="operationCardGB">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>451</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="getSumBtn">
          <property name="text">
           <string>Состояния баланса</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="sumDisplay">
          <property name="text">
           <string>0, руб</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="addSumBtn">
          <property name="text">
           <string>Пополнить баланс</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sumLine">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Активация(Деактивация)</string>
     </attribute>
     <widget class="QLabel" name="statusActivationDisplay">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>196</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="deactivateBtn">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>54</y>
        <width>197</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Деактивация карты</string>
      </property>
     </widget>
     <widget class="QPushButton" name="activateBtn">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>20</y>
        <width>197</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Активация новой карты</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusActivationDisplay_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>196</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QPushButton" name="ClearLogBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистка лога</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>231</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="ConnectBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Подключить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DisconectBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Отключить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>53</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="SerchBtn">
         <property name="text">
          <string>Поиск порта</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="PortNameBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="BaudRateBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="ClearLogBtn_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить настройки</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>231</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Автоматические подключение к порту</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="testTab">
     <attribute name="title">
      <string>Тестирования</string>
     </attribute>
     <widget class="QGroupBox" name="testGB">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>441</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Режим тестирования</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="SendDataBtn">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Отправить данные</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="send_data">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ClearLogBtn</sender>
   <signal>clicked()</signal>
   <receiver>consol</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
